tools/hifiasm/hifiasm.xml

```diff
@@ -125,6 +134,16 @@
             --l-msjoin $hic_partition.l_msjoin
         #end if
 
+        #if str($ont_integration.ont_integration_selector) == 'set':
+            --ul '${ ' '.join(["%s" % $x for $x in $ultralong_inputs]) }'
```

Comment:
For your other question, I think this here is wrong - at least in combination with the outer `'`.

So what is happening here. the join is creating a string like `foo bar baz` and then you are putting `'` around it - yielding to `'foo bar baz'`. But you want `'foo' 'bar' 'baz'`.

Try something like that ... note the double inner quotes and the removed outer quotes.

`  ${ ' '.join(["'%s'" % $x for $x in $ultralong_inputs]) }  `

Reply:
<img width="1087" alt="image" src="https://user-images.githubusercontent.com/48773001/214348834-c916c9ed-e86d-465d-b5ce-1b857555ad32.png">
looks good! should i change the syntax on the HiC partition section to match as well?

Reply:
Yes please!